README.adoc

```diff
@@ -454,6 +455,48 @@ tomcat:
 <4> Optional parameter to configure sleeping time between log downloading. You can set in seconds using _s_ or miliseconds using _ms_. By default time unit is miliseconds and value 500.
 <5> Optional parameter to configure number of retries to be done. By default 10 iterations are done.
 
+Custom Await strategy:
+
+[source, java]
+.org.arquillian.cube.docker.impl.await.CustomAwaitStrategyImpl
+----
+public class CustomAwaitStrategyImpl implements AwaitStrategy {
+
+  Await params;
+  DockerClientExecutor dockerClientExecutor;
+  Cube<?> cube;
+
+  public void setCube(Cube<?> cube) {
```

Comment:
I thought the same butI don't like to provide two ways of doing the same thing, or for example you need to check the permutations of all of them since not all the parameters might be mandatory. For example:

```
public Custom(Cube, DockerClient, Await)

or

public Custom(Cube, DockerClient)

or 

public Custom( DockerClient, Await)

and so on
```